daemon/graphdriver/aufs/aufs.go

```diff
@@ -579,23 +561,20 @@ func (a *Driver) mounted(mountpoint string) (bool, error) {
 
 // Cleanup aufs and unmount all mountpoints
 func (a *Driver) Cleanup() error {
-	var dirs []string
-	if err := filepath.Walk(a.mntPath(), func(path string, info os.FileInfo, err error) error {
```

Comment:
😱

Reply:
We have something similar in devmapper, too...